diff --git a/packages/components/components/drawer/DrawerSidebar.tsx b/packages/components/components/drawer/DrawerSidebar.tsx
--- a/packages/components/components/drawer/DrawerSidebar.tsx
+++ b/packages/components/components/drawer/DrawerSidebar.tsx
@@ -34,7 +34,7 @@ const DrawerSidebar = ({ buttons }: Props) => {
     return (
         <nav
             aria-label={c('Landmarks').t`Side panel`}
-            className={clsx('drawer-sidebar hidden md:inline no-print', isProminent && 'ui-prominent')}
+            className={clsx('drawer-sidebar inline no-print', isProminent && 'ui-prominent')}
         >
             <span className="flex flex-column items-center py-3 h-full">
                 <div className="flex flex-column items-center gap-5">{clonedButtons}</div>
diff --git a/packages/components/components/drawer/DrawerVisibilityButton.tsx b/packages/components/components/drawer/DrawerVisibilityButton.tsx
--- a/packages/components/components/drawer/DrawerVisibilityButton.tsx
+++ b/packages/components/components/drawer/DrawerVisibilityButton.tsx
@@ -39,7 +39,7 @@ const DrawerVisibilityButton = ({ spotlightSeenRef }: Props) => {
         <Tooltip title={tooltipText} originalPlacement="left">
             <span
                 className={clsx(
-                    'drawer-visibility-control hidden md:flex',
+                    'drawer-visibility-control flex',
                     showDrawerSidebar
                         ? 'drawer-visibility-control--hide'
                         : 'drawer-visibility-control--show ui-standard'
